rspec-mode.el

```diff
@@ -255,6 +255,13 @@ for spec files corresponding to files inside them."
   :safe 'listp
   :group 'rspec-mode)
 
+(defcustom rspec-unit-test-dir nil
```

Comment:
Should this really be called "-unit-"? People write all kinds of specs with RSpec.